content/_data/upgrades/2-303-1.adoc

```diff
@@ -0,0 +1,176 @@
+The transition looks like this:
+
+* 2.303.1-lts is replaced with 2.303.1
+* 2.303.1-lts-alpine is replaced with 2.303.1-alpine
+* 2.303.1-lts-centos7 is replaced with 2.303.1-centos7
+* 2.303.1-lts-slim is replaced with 2.303.1-slim
+
+==== Digester 2 removed
+
+The Apache Commons Digester v2.1 has been removed from Jenkins core.
+The library was last released in 2010.
+Plugins that depend on the library need to be updated to remove that dependency.
+
+Plugins that are affected include:
+
+* link:https://plugins.jenkins.io/vs-code-metrics/[Visual Studio Code Metrics]
+* link:https://plugins.jenkins.io/BlameSubversion/[Blame Subversion]
+* link:https://plugins.jenkins.io/javatest-report/[JavaTest report]
+* link:https://plugins.jenkins.io/vss/[Visual SourceSafe]
+* link:https://plugins.jenkins.io/synergy/[Synergy]
+* link:https://plugins.jenkins.io/config-rotator/[Config Rotator]
+* link:https://plugins.jenkins.io/harvest/[Harvest SCM]
+* link:https://plugins.jenkins.io/cmvc/[CMVC]
+
+Suspended plugins (no longer distributed by the Jenkins update center) that are affected include:
+
+* link:https://plugins.jenkins.io/tfs/[TFS]
+* link:https://issues.jenkins-ci.org/browse/INFRA-2487[svn-release-mgr]
+* link:https://plugins.jenkins.io/cpptest/[cpptest]
+* link:https://plugins.jenkins.io/CFLint/[CFLint]
+
+==== Stop bundling External Monitor Job Type, LDAP, and PAM plugins with Jenkins
+
+Certain key Jenkins capabilities were initially created inside the Jenkins core.
+As Jenkins has developed further, capabilities moved from inside Jenkins core to dedicated Jenkins plugins, like the plugin:external-monitor-job[External Monitor Job plugin], the plugin:ldap[LDAP plugin], and the plugin:pam[PAM authentication plugin].
+When those plugins were created, the plugins were "bundled" inside the jenkins.war file to retain compatibility for plugins that depended on their functionality to be inside Jenkins core.
+The plugin:external-monitor-job[External Monitor Job plugin], the plugin:ldap[LDAP plugin], and the plugin:pam[PAM authentication plugin] are no longer bundled with Jenkins.
+
+In very rare cases, this could result in problems when attempting to install plugins compatible with Jenkins (then Hudson) before 1.467.
+If you use a plugin that relies on the functionality provided by the plugin:external-monitor-job[External Monitor Job plugin], the plugin:ldap[LDAP plugin], or the plugin:pam[PAM authentication plugin] and manage plugins outside the Jenkins plugin manager, you will now need to ensure yourself that a recent release of those plugins are installed.
+Jenkins will attempt to load such plugins but may fail at any time during startup or afterwards with `ClassNotFoundException` or similar.
+In these cases, the best path forward is usually to modify the plugin that has the problem so that it requires a newer version of Jenkins core.
+
+==== Removed JEP-200 compatibility workarounds
+
+JEP-200 workarounds for several plugins have been removed.
```

Comment:
Add a link and summarize what it is about?

Reply:
Good suggestion.  I added a brief description of JEP-200 with a link to the JEP-200 LTS blog post and a link to JEP-200 itself.  The reader can follow a link in the LTS blog post to the original JEP-200 blog post.

See 91cef558e331ad6470b8f4af5385cdb8f1a84cc4